frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/components/uploadListItemDestination/uploadListItemDestination.component.tsx

```diff
@@ -244,4 +235,4 @@ export const UploadListItemDestination = ({
 			)}
 		/>
 	);
-};
+}, (prev, next) => prev.revisionPrefix === next.revisionPrefix && prev.value === next.value && prev.disabled === next.disabled);
```

Comment:
Is the memoisation here actually doing a difference?

Reply:
I think controller can pass extra stuff that we dont care about so, for now yes. I think controller can be spared, but I wouldnt modify Atm